analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Utilities/SymbolReferenceAnalyzer/ToDo.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace TestCases;
```

Comment:
Useful to assert how the symbols are correctly computed when the types are defined in another file.